selfdrive/car/interfaces.py

```diff
@@ -217,9 +218,10 @@ def _update(self, c: car.CarControl) -> car.CarState:
 
   def update(self, c: car.CarControl, can_strings: list[bytes]) -> car.CarState:
     # parse can
+    can_list = can_capnp_to_list(can_strings)
```

Comment:
because only one conversion (capnp->list) needs to be done.  the overall performance will be better than the previous version of opendbc.

Reply:
so it's not twice as slow?

Reply:
updated the test case to remove the overhead of conversion: https://github.com/commaai/openpilot/pull/32048
the new test results is:

before:

> 343.58 mean ms, 349.11 max ms, 339.32 min ms, 3.40 std ms

after:

> 476.56 mean ms, 498.29 max ms, 464.91 min ms, 10.96 std ms

It's still a little slower, but not twice as much.

Reply:
hmm, even the old speed seems a bit slow. any other optimizations we can do at least get it to be the same?

Reply:
I'll thinking of ways to optimize further.

Reply:
Submitted a PR https://github.com/commaai/opendbc/pull/1039 to improve the performance of `update_string`, now it's twice as fast. This PR will also benefit from it once merged.